staging/src/k8s.io/client-go/transport/token_source_test.go

```diff
@@ -140,12 +140,23 @@ func TestCachingTokenSourceRace(t *testing.T) {
 
 		var wg sync.WaitGroup
 		wg.Add(100)
-
+		errMsgCh := make(chan string, 1)
```

Comment:
this was already addressed in https://github.com/kubernetes/kubernetes/pull/100429